extensions/control-plane/provision/provision-gcs/src/main/java/org/eclipse/edc/connector/provision/gcp/GcsProvisioner.java

```diff
@@ -89,28 +103,23 @@ public CompletableFuture<StatusResult<ProvisionResponse>> provision(
         }
     }
 
+    private String getServiceAccountName(GcsResourceDefinition resourceDefinition) {
+        if (resourceDefinition.getServiceAccountName() != null) {
+            // TODO verify service account name from resource definition before returning.
```

Comment:
is this TODO supposed to be solved in this PR or is there an issue related to it?

Reply:
This TODO requires more internal sync, there is currently no issue related but I would target this in another PR